vividus/src/main/java/org/vividus/softassert/AssertionManager.java

```diff
@@ -27,6 +27,7 @@
     private final ISoftAssert softAssert;
 
     private boolean failFast;
+    private boolean knownIssueFailFast;
```

Comment:
I believe it would be better to have the ability to specify which known-issue should fail-fast scenario and which shouldn't